Tests/DuplicatedSDKTest/DuplicatedSDKTest/DuplicatedSDKTestApp.swift

```diff
@@ -0,0 +1,50 @@
+import ObjectiveC.runtime
+import Sentry
+import SwiftUI
+
+@main
+struct DuplicatedSDKTestApp: App {
+    
+    init () {
+        SentrySDK.start { options in
+            options.dsn = "https://a@o.sentry.io/1"
```

Comment:
`m`: What drove you to not put the DSN of the other sample apps here?

Reply:
Actually it was copilot auto complete that wrote that. And since it works, I see no reason to use a valid one.

Reply:
If somebody once debugs it manually, you could expect that the project also sends the events to sentry-cocoa in the sentry-sdks org, as every other sample project does. If it doesn't, it could take a while to figure this out.

Reply:
You may be overthinking.  That is not the purpose of this test project, but it is also fine; it does not really matter.
```suggestion
            options.dsn = "https://6cc9bae94def43cab8444a99e0031c28@o447951.ingest.sentry.io/5428557"
```